doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -36,13 +36,44 @@ one backend layer.
 .. image:: ec_host_cmd_shi.png
    :align: center
 
+Another case is SPI. Unfortunately, the current SPI API can't be used to handle the host commands
+communication. The main issues are unknown command size sent by the host (the SPI transaction
+sends/receives specific number of bytes) and need to constant sending status byte (the SPI module
+is enabled and disabled per transaction). It forces implementing the SPI driver within a backend,
+as it is done for SHI. That means a SPI backend has to implemented per chip family. However, it
+can be changed in the future once the SPI API is extended to host command needs.
+
+That approach requires configuring the SPI dts node in a special way. The compatible property of a
+SPI node has changed to use the Host Command version of a SPI driver. The rest of the properties
+should be configured as usual. Example of the SPI node for STM32:
+
+.. code-block:: devicetree
+
+   &spi1 {
+           /* Change the compatible string to use the Host Command version of the
+            * STM32 SPI driver
+            */
+           compatible = "st,stm32-spi-host-cmd";
```

Comment:
I'd suggest to handle this slightly differently and use:
```
			compatible = "st,stm32h7-spi", "st,stm32-spi-fifo", "st,stm32-spi-host-cmd";
```
This way, you keep the benefit of using "st,stm32h7-spi" and "st,stm32-spi-fifo" which are currently used to specify variants of STM32 SPI HW block.

I acknowledge this is less easy to use, but this will ease the implementation of this driver on multiple STM32 targets.
You can think or other ways to do it of course, but I think it would be easier for maintenance if you stay in sync with "main" STM32 SPI driver on that regard.

Reply:
Please tell me if I understand it correctly.

The `"st,stm32-spi"`  compatible string is always use for STM32 SPI block. The `"st,stm32h7-spi"` can be added as a second compatible string to specify variant of the SPI block that works a bit differently.  `"st,stm32-spi-fifo"` is the same case.

So, in this case it is better just to change the `"st,stm32-spi"` string to `"st,stm32-spi-host-cmd"` and use the "additional" ones (`"st,stm32h7-spi"` and `"st,stm32-spi-fifo") as usual, because it doesn't cause building the  standard driver anyway.

Is that correct? Thanks in advice.

Reply:
This is fully correct

Fyi, spi_ll_stm32.c file is built in based on CONFIG_SPI_STM32=y, which is selected based on `st,stm32-spi` compat being enabled:
```
menuconfig SPI_STM32
	bool "STM32 MCU SPI controller driver"
	default y
	depends on DT_HAS_ST_STM32_SPI_ENABLED
```

Reply:
Great, working on the change.

Reply:
The example in the docs is still correct, right?